backend/src/routes/courses/CoursesRouter.js

```diff
@@ -5,7 +5,7 @@ var connection = require('../../../database/database')
 
 router.get('/', (req, res) => {
   connection.connect(err => {
-    const getCourses = `SELECT * FROM course WHERE Is_Active=TRUE`
+    const getCourses = `SELECT * FROM course`
```

Comment:
Hi Caleb, I think the get courses should only get the ones where the course is active. Can you help to add a WHERE clause for when Course_Status="Active"

Thanks :o